404.html

```diff
@@ -0,0 +1,27 @@
+---
+layout: default
+title: 404
+---
```

Comment:
Needs the `permalink: /404.html` front-matter, or the output file will be plunked into a `404/index.html` file which won't be picked up automagically by GitHub Pages (at least I don't think it will, better not to take the chance).

Reply:
Added!